template/partials/report.js

```diff
@@ -1,61 +1,99 @@
 'use strict'
 const prettyBytes = require('pretty-bytes')
 
+function minTwoDigits (n) {
+  return (n < 10 ? '0' : '') + n
+}
+
+function datestuff (date) {
+  date = new Date(date)
+  /* date.getMonth() + ' ' + date.getDate() + ', ' + date.getHours() + ':' + date.getMinutes() + ':' + date.getSeconds()
+*/
+  var temp1 = date.toDateString().substring(4)
+  var temp2 = temp1.substring(0, temp1.length - 4)
+  temp2 = temp2.trim()
+  var temp3 = temp2 + ', ' + minTwoDigits(date.getHours()) + ':' + minTwoDigits(date.getMinutes()) + ':' + minTwoDigits(date.getSeconds())
+  var ret = temp3
+  return ret
+}
```

Comment:
I think date can be simplified, lets add npm.im/moment for this.

If you do `npm install moment --save`, and then change this return to `return moment(date).format('MMMM Do YYYY, h:mm:ss a'` it should be really clean